platforms/corda-ent/configuration/roles/delete/vault_secrets/tasks/main.yaml

```diff
@@ -109,32 +109,32 @@
 # This task deletes crypto material
 - name: Delete Crypto material 
   shell: |
-    vault delete secret/{{ org.name | lower }}/certs/customnodekeystore
-    vault delete secret/{{ org.name | lower }}/certs/{{ network | json_query('orderers[?type==`idman`].name') | first }}
-    vault delete secret/{{ org.name | lower }}/certs/{{ network | json_query('orderers[?type==`networkmap`].name') | first }}
-    vault delete secret/{{ org.name | lower }}/certs/networkmaptruststore
-    vault delete secret/{{ org.name | lower }}/certs/nodekeystore
-    vault delete secret/{{ org.name | lower }}/certs/sslkeystore
-    vault delete secret/{{ org.name | lower }}/certs/truststore
-    vault delete secret/{{ org.name | lower }}/tlscerts
-    vault delete secret/{{ org.name | lower }}/certs
-    vault delete secret/{{ org.name | lower }}/root/certs
-    vault delete secret/{{ org.name | lower }}/network-parameters
+    vault delete secret/{{ org.name | lower }}/{{ peer.name | lower }}/certs/customnodekeystore
+    vault delete secret/{{ org.name | lower }}/{{ peer.name | lower }}/certs/{{ network | json_query('orderers[?type==`idman`].name') | first }}
+    vault delete secret/{{ org.name | lower }}/{{ peer.name | lower }}/certs/{{ network | json_query('orderers[?type==`networkmap`].name') | first }}
+    vault delete secret/{{ org.name | lower }}/{{ peer.name | lower }}/certs/networkmaptruststore
+    vault delete secret/{{ org.name | lower }}/{{ peer.name | lower }}/certs/nodekeystore
+    vault delete secret/{{ org.name | lower }}/{{ peer.name | lower }}/certs/sslkeystore
+    vault delete secret/{{ org.name | lower }}/{{ peer.name | lower }}/certs/truststore
+    vault delete secret/{{ org.name | lower }}/{{ peer.name | lower }}/tlscerts
+    vault delete secret/{{ org.name | lower }}/{{ peer.name | lower }}/certs
+    vault delete secret/{{ org.name | lower }}/{{ peer.name | lower }}/root/certs
+    vault delete secret/{{ org.name | lower }}/{{ peer.name | lower }}/network-parameters
```

Comment:
This needs to be updated in the docs as well

Reply:
Is part of documentation changes I am working now.

Reply:
ok